### PR TITLE
add required style for object and array editor

### PR DESCRIPTION
**Pull request by Danielecina**
#### Info
there is no need to wait for alignment with json-editor

#### 📹  video 

requiredFieldsStyleForArray Object

#### 📄  Checklist

- [x] your branch will not cause merge conflict with `master`
- [x] run `npm test`
- [x] tests are included
- [ ] the documentation is updated or integrated accordingly with your changes
- [x] the code will follow the lint rules and style of the project
- [x] you are not committing extraneous files or sensible data

### CODE REVIEW

File: packages/api-explorer/package-lock.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
 	"name": "@mia-platform/api-explorer",
-	"version": "6.4.0",
+	"version": "6.5.2",
```
Comment (fredmaggiowski) [inline packages/api-explorer/package-lock.json:3]: Why this change?

**Review by fredmaggiowski — APPROVED** [review]

File: packages/api-explorer/package-lock.json (hunk 1 above)
Comment (Danielecina) [reply packages/api-explorer/package-lock.json:3]: my error, sorry

File: packages/api-explorer/src/JsonForm/arrays-editor.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+module.exports = (classReference) => class arrayCustomEditor extends classReference {
+
+  postBuild() {
+    super.postBuild()
+
+    if (this.parent) {
+      this.header.style.textTransform = 'unset'
```
Comment (dibericky) [inline packages/api-explorer/src/JsonForm/arrays-editor.js:7]: `this.header` is *always* existing?

Comment (dibericky) [reply packages/api-explorer/src/JsonForm/arrays-editor.js:7]: added check just to be sure

**Review by dibericky — APPROVED** [review]
